cvat-ui/src/actions/shortcuts-actions.ts

```diff
@@ -3,16 +3,21 @@
 //
 // SPDX-License-Identifier: MIT
 
+import { KeyMapItem } from 'utils/mousetrap-react';
 import { ActionUnion, createAction } from 'utils/redux';
 
 export enum ShortcutsActionsTypes {
     SWITCH_SHORTCUT_DIALOG = 'SWITCH_SHORTCUT_DIALOG',
+    REGISTER_SHORTCUT = 'REGISTER_SHORTCUT',
```

Comment:
As this action allows to register a list of shortcuts: 

```suggestion
    REGISTER_SHORTCUTS = 'REGISTER_SHORTCUTS',
```